src/mono/wasm/runtime/logging.ts

```diff
@@ -148,4 +148,8 @@ export function parseSymbolMapFile(text: string) {
     });
 
     mono_log_debug(`Loaded ${wasm_func_map.size} symbols`);
+}
+
+export function mono_wasm_get_func_id_to_name_mappings() {
+    return [...wasm_func_map.values()];
```

Comment:
Could this rather return URL of the symbol files ? Or perhaps it's we could teach DebugProxy to read the blazor.boot.json and resolve .PDBs and DLLs in the same way too ?

Reply:
In Net9 maybe